test/OpenTelemetry.Exporter.Geneva.Tests/OtlpProtobufMetricExporterTests.cs

```diff
@@ -69,27 +93,86 @@ public OtlpProtobufMetricExporterTests()
         this.TagList.Add(new("uintKey", uintValue));
         this.TagList.Add(new("ulongKey", ulongValue));
         this.TagList.Add(new("ushortKey", ushortValue));
+
+        this.exemplarTagList = this.TagList;
+        this.exemplarTagList.Add(new("zfilteredKey1", "zfilteredValue1"));
     }
 
     [Theory]
-    [InlineData("longcounter", 123L, null, true, true)]
-    [InlineData("longcounter", 123L, null, true, false)]
-    [InlineData("longcounter", 123L, null, false, true)]
-    [InlineData("longcounter", 123L, null, false, false)]
-    [InlineData("doublecounter", null, 123.45, true, true)]
-    [InlineData("doublecounter", null, 123.45, true, false)]
-    [InlineData("doublecounter", null, 123.45, false, true)]
-    [InlineData("doublecounter", null, 123.45, false, false)]
-    [InlineData("longcounter", -123L, null, true, true)]
-    [InlineData("longcounter", -123L, null, true, false)]
-    [InlineData("longcounter", -123L, null, false, true)]
-    [InlineData("longcounter", -123L, null, false, false)]
-    [InlineData("doublecounter", null, -123.45, true, true)]
-    [InlineData("doublecounter", null, -123.45, true, false)]
-    [InlineData("doublecounter", null, -123.45, false, true)]
-    [InlineData("doublecounter", null, -123.45, false, false)]
-    public void CounterSerializationSingleMetricPoint(string instrumentName, long? longValue, double? doubleValue, bool addPrepopulatedDimensions, bool addAccountAndNamespace)
+    [InlineData("longcounter", 123L, null, true, true, true, true)]
```

Comment:
This is huge number of test cases for `InlineData` based approach. You might want to check if there's a better way to test these cases. Maybe using xUnit ClassData? https://andrewlock.net/creating-parameterised-tests-in-xunit-with-inlinedata-classdata-and-memberdata/

@CodeBlanch Any suggestions?

Reply:
Yeah, this probably needs some rewrite. I was considering to decouple some testing in new tests. It's the number of possible combinations that is making this big.

Reply:
It's probably okay to remove the combinations for account and namespace for exemplar related tests.